main/http_server/axe-os/src/app/components/settings/settings.component.ts

```diff
@@ -44,12 +44,21 @@ export class SettingsComponent {
 
     window.addEventListener('resize', this.checkDevTools);
     this.checkDevTools();
+    this.info$ = this.systemService.getInfo().pipe(shareReplay({refCount: true, bufferSize: 1}))
 
-    this.latestRelease$ = this.githubUpdateService.getReleases().pipe(map(releases => {
-      return releases[0];
-    }));
+    this.latestRelease$ = of(null).pipe(
```

Comment:
A slightly cleaner way:

this.latestRelease$ = defer(() =>
  this.ASICModel
    ? this.githubUpdateService.getReleases(this.ASICModel).pipe(
        map(releases => releases[0])
      )
    : of(null)
);